extensions/amp-ad-network-doubleclick-impl/0.1/amp-ad-network-doubleclick-impl.js

```diff
@@ -1298,14 +1298,16 @@ export class AmpAdNetworkDoubleclickImpl extends AmpA4A {
         'gutData': JSON.stringify(dict({
           'events': [{
             'timestamp': Date.now(),
-            'slotid': this.troubleshootData_.slotId,
+            'slotid': this.troubleshootData_.slotId + '_' +
```

Comment:
Let's make slotId a const var and reference accordingly:

const slotId = this.troubleshootData_.slotId + '_' + this.troubleshootData_.slotIndex;
const payload = dict({... slotid: slotId...

Reply:
Fixed.